src/__tests__/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
-  "extends": "../tsconfig",
+  "extends": "../../tsconfig",
   "compilerOptions": {
     "declaration": false
   },
   "exclude": [],
-  "include": ["./**/*.ts"]
+  "include": ["**/*.test.(ts|tsx)"]
```

Comment:
Removing the `.test` from the files names was explicit in the `kcd-scripts` migration (#502), but I'm no bothered and happy to keep them if you prefer it.